examples/csharp/HelloworldLegacyCsproj/Greeter/Greeter.csproj

```diff
@@ -37,7 +37,7 @@
       <HintPath>..\packages\Google.Protobuf.3.6.1\lib\net45\Google.Protobuf.dll</HintPath>
     </Reference>
     <Reference Include="Grpc.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=d754f35622e28bad, processorArchitecture=MSIL">
-      <HintPath>..\packages\Grpc.Core.1.17.0\lib\net45\Grpc.Core.dll</HintPath>
```

Comment:
Have you tried running the examples? It seems odd that Grpc.Core.Api is missing here.